src/collector/views.py

```diff
@@ -53,6 +53,24 @@ def findImage(url):
 			return turl
 		turl = "https://i.imgur.com" + urlparse(url).path + ".jpg"
 		res = requests.head(turl)
+		if res.status_code == 200:
+			return turl
+		turl = "https://i.imgur.com" + urlparse(url).path + ".jpeg"
+		res = requests.head(turl)
+		if res.status_code == 200:
+			return turl
+		return None
+	elif host in ["gayazo.com"]:
```

Comment:
Shouldn't this be gyazo? https://github.com/Corridors-of-Time-Transcription/puzzlepieces/issues/55

Reply:
absolutely correct...